src/io/vtk/vtk-custom-decimal.h

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file vtk-custom-decimal.h
+ * @brief Required for mapping the 'decimal' data type to 'long double' (would be 'double' otherwise).
+ */
+
+#include <xsd/cxx/xml/string.hxx> // xml::transcode
+
+#include <xsd/cxx/tree/text.hxx>  // text_content
+
+using namespace std;
```

Comment:
is there any `xsdcxx` switch that removes this? Having this in a header is not cool :/